plonky2x/core/src/backend/prover/remote.rs

```diff
@@ -114,9 +122,18 @@ impl RemoteProver {
             .collect_vec();
         let (batch_id, proof_ids) = service.submit_batch(&requests)?;
 
-        const MAX_RETRIES: usize = 1500;
-        for _ in 0..MAX_RETRIES {
-            sleep(Duration::from_secs(10)).await;
+        // Default timeout for a batch proof is 60 minutes. Users can override this value by
+        // setting the BATCH_PROOF_TIMEOUT_MINS environment variable.
+        const DEFAULT_BATCH_PROOF_TIMEOUT_MINS: u64 = 60;
+        let proof_timeout_mins = env::var("BATCH_PROOF_TIMEOUT_MINS")
```

Comment:
nit: atleast in Go, it is better to read env at top-level and pass it into this function as an argument. This way you can error immediately if .env is incorrect (I guess with defaulting it might be fine)

Reply:
Yeah I agree if it's required, but because we have defaults, I'll keep it as this for now. As there are no default args in Rust, probably better to do it this way too.